docs/en/operation/User-CLI.md

```diff
@@ -813,10 +813,12 @@ $ ./bin/alluxio fs cp /hdfs/file1 /s3/
 ### distributedCp
 
 The `distributedCp` command copies a file or directory in the Alluxio file system distributed across workers
-using the job service. In sync mode which is the default mode, user will get a JOB_CONTROL_ID after the command submits successfully.
-User need to wait for the command to finish. When the command finishes, user will be able to see the file copy status and statistics on completion/failure in sync mode.
-In async mode user will get a JOB_CONTROL_ID after the command submission is successful. The command execution happens asynchronously on the Job Master side.
-User can use `getCmdStatus` to check detailed status information on the JOB_CONTROL_ID.
+using the job service. By default, the command runs synchronously and the user will get a `JOB_CONTROL_ID` after the command successfully submits the job to be executed.
+The command will wait until the job is complete, at which point the user will see the list of files copied and statistics on which files completed or failed.
+The command can also run in async mode with the `--async` flag. Similar to before, the user will get a `JOB_CONTROL_ID` after the command successfully submits the job.
+The difference is that the command will not wait for the job to finish. 
+Users can use the [`getCmdStatus`](#getCmdStatus) [`getCmdStatus`]({{ '/en/operation/User-CLI.html' | relativize_url }}#getCmdStatus)
```

Comment:
only provide one, the current state will actually show up with two words with different links pointing to the same thing
```suggestion
Users can use the [`getCmdStatus`](#getCmdStatus)
```
the example i gave can also be used for anchor tags in different pages, but since it is in the same page the shorter version is preferred